app/src/main/java/org/p2p/wallet/user/repository/prices/impl/TokenPricesCoinGeckoRepository.kt

```diff
@@ -36,6 +36,13 @@ class TokenPricesCoinGeckoRepository(
         loadPrices(tokenSymbols, targetCurrency)
     }
 
+    override suspend fun getTokenPriceBySymbol(
+        tokenSymbol: TokenSymbol,
+        targetCurrency: String
+    ): TokenPrice {
+        return loadPrices(listOf(tokenSymbol), targetCurrency).first()
```

Comment:
Are we able to load a price for a single token?

Reply:
Actually there are methods for single prices, I just used the implemented ones

Reply:
Actually, you're right. If we're sending request for one token price, why not requesting all and cache them before home

Reply:
But if we want to load and cache all user tokens prices on DerivableAccount we should make 2 bigger requests and make caching job, screen could be overloaded